plugins/modules/managed_serviceaccount_rbac_addon.py

```diff
@@ -0,0 +1,263 @@
+#!/usr/bin/python
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+
+module: managed_serviceaccount_rbac_addon
+
+short_description: cluster proxy addon
+
+author:
+- "Hao Liu (@TheRealHaoLiu)"
+- "Hanqiu Zhang (@hanqiuzh)"
+- "Nathan Weatherly (@nathanweatherly)"
+- "Tsu Phin Hee (@tphee)"
+
+description:
+- Use the managed-serviceaccount RBAC addon to setup a serviceaccount on a managedcluster with the specified RBAC permission,
+    and return the serviceaccount token.
+
+options:
+    hub_kubeconfig:
+        description: Path to the Hub cluster kubeconfig. Can also be specified via K8S_AUTH_KUBECONFIG environment variable.
+        type: str
+        required: True
+    managed_cluster:
+        description: Name of the managed cluster.
+        type: str
+        required: True
+    managed_serviceaccount:
+        description: Managed serviceaccount information.
+        type: dict
+        required: True
+    rbac_template:
+        description: Path to the RBAC template file.
```

Comment:
may be we can add some more details here that it can be rolebinding only, or role+rolebinding?